website/docs/r/advanced_cluster.html.markdown

```diff
@@ -446,9 +446,9 @@ Include **desired options** within advanced_configuration:
  }
 ```
 
-* `default_read_concern` - (Optional) [Default level of acknowledgment requested from MongoDB for read operations](https://docs.mongodb.com/manual/reference/read-concern/) set for this cluster. MongoDB 4.4 clusters default to [available](https://docs.mongodb.com/manual/reference/read-concern-available/).
+* `default_read_concern` - **(DEPRECATED)** (Optional) [Default level of acknowledgment requested from MongoDB for read operations](https://docs.mongodb.com/manual/reference/read-concern/) set for this cluster. MongoDB 4.4 clusters default to [available](https://docs.mongodb.com/manual/reference/read-concern-available/).
```

Comment:
was thinking if we want to add the reasoning behind the deprecation in the documentation, or only leave this in the migration guide.

Reply:
I think we could consider adding a link to the migration guide, but would not prefer to add reasoning to avoid redundancy.

Reply:
what about at least "what should they do for that field" here in the doc?

Reply:
added guidance wherever possible